glustercli/cmd/volume-create.go

```diff
@@ -166,26 +166,44 @@ func volumeCreateCmdRun(cmd *cobra.Command, args []string) {
 	if flagCreateReplicaCount > 0 {
 		// Replicate Volume Support
 
-		if numBricks%flagCreateReplicaCount != 0 {
-			failure("Invalid number of bricks specified", nil, 1)
+		// if arbiter count is specified
+		if cmd.Flags().Changed("arbiter") {
+			if flagCreateArbiterCount != 1 && flagCreateReplicaCount != 2 {
+				failure("Invalid arbiter/replica count specified. Supported arbiter configuration: replica 2, arbiter 1", nil, 1)
+			}
+		}
+
+		if numBricks%(flagCreateReplicaCount+flagCreateArbiterCount) != 0 {
+			failure("Invalid number of bricks specified. Number of bricks must be a multiple of replica (+arbiter) count.", nil, 1)
 		}
 
-		numSubvols := numBricks / flagCreateReplicaCount
+		// FIXME: Things would've been so much simpler if specifying the
+		// arbiter brick was simply a separate command flag such as
+		// --arbiter-brick or even a boolean as we always pick the last
+		// brick as arbiter. Backward compatibility is a pain and we're
+		// doing the same bad choices all over again.
```

Comment:
I think it is not only for backward compatibility, but it is for future extension. In future, if we support `replica 3` and `arbiter 2`. 

@itisravi please add your thoughts.